src/authentication/components/Subscription/SubscriptionOptions.tsx

```diff
@@ -0,0 +1,133 @@
+import * as React from 'react'
+import { UserSubscription } from 'src/authentication/components/user-subscription'
+import Button from 'src/popup/components/Button'
+import { Helmet } from 'react-helmet'
+import { SubscriptionPriceBox } from 'src/authentication/components/Subscription/SubscriptionPriceBox'
+import { AuthenticatedUser } from 'src/authentication/background/types'
+import { auth } from 'src/util/remote-functions-background'
+import { UserPlans } from 'firebase-backend/firebase/functions/src/types'
+const chargeBeeScriptSource = 'https://js.chargebee.com/v2/chargebee.js'
+
+//
+export const subscriptionConfig = {
+    site:
+        process.env.NODE_ENV !== 'production'
+            ? 'worldbrain-test'
+            : 'worldbrain',
+}
+
+interface Props {
+    user: AuthenticatedUser | null
+    onClose?: () => void
+    subscriptionChanged: () => void
+}
+export class SubscriptionOptions extends React.PureComponent<Props> {
```

Comment:
Does this work? It's a PureComponent, but if I'm correct `this.subscribed` will be mutated,